index.js

```diff
@@ -17,50 +17,56 @@ const normalizeEmitter = emitter => {
 	};
 };
 
-module.exports = (emitter, event, options) => {
+const multiple = (emitter, event, options) => {
```

Comment:
This method should type-check that the `count` option is set, that's it's a positive number, and that it's an integer, otherwise throw a TypeError. To prevent mistakes.